substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/debugentry/FileEntry.java

```diff
@@ -32,10 +32,12 @@
 public class FileEntry {
     private String fileName;
     private DirEntry dirEntry;
+    private int idx;
```

Comment:
Should we mark these fields as final?

Reply:
Yes will do.

Reply:
There are actually a lot of other fields of classes in the `debugentry` subpackage that can be final. I will work thorough them all systematically as part of the review rework. Also, the classes in the `dwarf` subpackage

Reply:
Done

Reply:
Thank you!